projects/arlas-toolkit/src/lib/components/config-manager/config-menu/config-menu.component.ts

```diff
@@ -125,11 +135,44 @@ export class ConfigMenuComponent implements OnInit {
       }
       case ConfigActionEnum.DUPLICATE:
       case ConfigActionEnum.RENAME:
-      case ConfigActionEnum.SHARE: {
         if (action.config && action.config.id) {
           this.getDialogRef(action).subscribe(() => this.actionExecutedEmitter.next(action));
         }
         break;
+      case ConfigActionEnum.SHARE: {
+        if (action.config && action.config.id &&
+          action.config.value &&
+          JSON.parse(action.config.value).arlas &&
+          JSON.parse(action.config.value).arlas.server &&
+          JSON.parse(action.config.value).arlas.server.url) {
```

Comment:
JSON.parse(action.config.value).arlas?.server?.url would be more concise

Also, I think it is good practice when listing a combination of booleans like this, to put the operator at the beginning of the line